lib/src/components/NxPagination/NxPagination.tsx

```diff
@@ -38,18 +38,25 @@ function validate(pageCount: number, currentPage: number | null | undefined) {
 }
 
 /**
- * @return the index of the first page in the current page range
+ * @return a tuple containing the index of the first page in the current page range
+ * and the index one beyond that of the last page in this range
  */
-function getCurrentPageRangeStart(pageCount: number, currentPage: number) {
+function getCurrentPageRangeBounds(pageCount: number, currentPage: number) {
```

Comment:
These changes were necessary to get the correct ARIA attributes on the last page button in the 5n+1 special case